src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -112,6 +112,8 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "beautifulsoup4": ("bs4",),
     "bitvector": ("BitVector",),
     "cattrs": ("cattr", "cattrs"),
+    "databricks-sdk": ("databricks",),
+    "databricks-sql-connector": ("databricks",),
```

Comment:
I believe we want `("databricks.sdk",)` and `("databricks.sql",)`

Reply:
Yes we do! My apologies. I didn't fully understand this and now I grok it to a much deeper level. I've updated the PR.